drivers/SmartThings/jbl/src/log.lua

```diff
@@ -0,0 +1,56 @@
+-- Copyright 2021 SmartThings
+--
+-- Licensed under the Apache License, Version 2.0 (the "License");
+-- you may not use this file except in compliance with the License.
+-- You may obtain a copy of the License at
+--
+--     http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing, software
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+
+-- luacheck: globals _envlibrequire
+
+local original_log
+if _envlibrequire ~= nil then
+  original_log = _envlibrequire("log")
+else
+  original_log = require("stdout_log")
+end
+
+local hubcore_enabled_log = {}
+
+hubcore_enabled_log.info = function(...)
+  original_log.info_with({hub_logs = true}, ...)
+end
+
+hubcore_enabled_log.warn = function(...)
+  original_log.warn_with({hub_logs = true}, ...)
+end
+
+hubcore_enabled_log.error = function(...)
+  original_log.error_with({hub_logs = true}, ...)
+end
```

Comment:
This seems like it would always be pushing data into the hub-core logs which I don't think we want to do.

Reply:
I suggest limiting to only warn/error going to hub logs, and only if we think that is needed.

Reply:
remove other log levels except warn and error